finetuner/client/client.py

```diff
@@ -92,28 +96,26 @@ def get_run(self, experiment_name: str, run_name: str) -> dict:
         )
         return self._handle_request(url=url, method=GET)
 
-    def list_runs(self, experiment_name: Optional[str] = None) -> List[dict]:
+    def list_runs(
+        self, experiment_name: Optional[str] = None, size: int = 50
+    ) -> List[Dict[str, Any]]:
         """List all created runs inside a given experiment.
 
         If no experiment is specified, list runs for all available experiments.
         :param experiment_name: The name of the experiment.
+        :param size: Number of runs to retrieve.
         :return: List of all runs.
```

Comment:
this might also be missleading. It is a list of pages of runs or something like this.